examples/traces/demo/src/index.php

```diff
@@ -1,7 +1,7 @@
 <?php
 
 declare(strict_types=1);
-require 'vendor/autoload.php';
+require '/../../../../vendor/autoload.php';
```

Comment:
This doesn't work, since the main libraries autoloader is out of the scope of the docker-compose context, and it would not pull in the dependencies of the demo (slim, ect.)

Reply:
Yup good call, switched it back.

I also was curious and tried to get this demo working, but got stuck after updating the tags on the opentelemetry-php-base images ("latest" was pulling in an image from 7 months ago) and exposing the 4317 port on the collector. 

Have a hunch something's off with the exporting from PHP but couldn't figure out how to troubleshoot (since this example wants to do all the config via environment variables and the OTEL_LOG_LEVEL one isn't supported yet afaict)

Reply:
Yap, I tried the demo as well a few days ago, and it didn't really work. However I used it about 2 months ago for a Otel workshop at work, and it worked fine.
Maybe @brettmc can have a loo at it, once he is back.

Reply:
I sure can take a look at it, right now.

Reply:
@Grunet @tidal I found a regression/bug. It's kinda hard to write a test for, since it involves a TracerProvider being destroyed as soon as it is created (I think introduced in the fix for #716 )
In short, this works:
```php
$tracerProvider = (new TracerProviderFactory('example'))->create();
$tracer = $tracerProvider->getTracer('io.opentelemetry.contrib.php');
```
and this doesn't work:
```php
$tracer = (new TracerProviderFactory('example'))->create()->getTracer('io.opentelemetry.contrib.php');
```

I'll take this up on slack...